validate/src/request.rs

```diff
@@ -117,6 +117,38 @@ pub const USERNAME_LIMIT_MAX: usize = 32;
 /// Minimum length of a username.
 pub const USERNAME_LIMIT_MIN: usize = 2;
 
+/// Maximum length of a webhook username.
+pub const WEBHOOK_USERNAME_LIMIT_MAX: usize = 80;
+
+/// Minimum length of a webhook username.
+pub const WEBHOOK_USERNAME_LIMIT_MIN: usize = 2;
+
+/// String value of an at sign.
+const AT_SIGN: &str = "@";
+
+/// String value of 'clyde'.
+const CLYDE: &str = "clyde";
+
+/// String value of a colon.
+const COLON: &str = ":";
+
+/// String value of 'discord'.
+const DISCORD: &str = "discord";
+
+/// String value of 'everyone'.
+const EVERYONE: &str = "everyone";
+
+/// String value of 'here'.
+const HERE: &str = "here";
+
+/// String value of an [octothorp].
+///
+/// [octothorp]: https://en.wikipedia.org/wiki/Number_sign#Names
+const OCTOTHORP: &str = "#";
+
+/// String value of three grave symbols.
+const THREE_GRAVES: &str = r#"```"#;
```

Comment:
Could we make this into two arrays of bad values? One with a list of values that can't be included in a given string and one of values that can't equal a given string? Or just one array with a tuple with a boolean denoting that?